easybuild/easyconfigs/f/FLASH/FLASH-2.2.00-foss-2020b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'FLASH'
+version = '2.2.00'
+
+homepage = 'https://ccb.jhu.edu/software/FLASH/'
+description = """FLASH (Fast Length Adjustment of SHort reads) is a very fast
+and accurate software tool to merge paired-end reads from next-generation
+sequencing experiments. FLASH is designed to merge pairs of reads when the
+original DNA fragments are shorter than twice the length of reads. The
+resulting longer reads can significantly improve genome assemblies. They can
+also improve transcriptome assembly when FLASH is used to merge RNA-seq data.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/dstreett/FLASH2/archive/']
+sources = ['%(version)s.zip']
+checksums = ['1e54b2dd7d21ca3e0595a3ffdd27ef3098f88c4de5b9302ec5ea074b49b79960']
+
+files_to_copy = [(['flash2'], 'bin')]
+
+postinstallcmds = ['ln -s %(installdir)s/bin/flash2 %(installdir)s/bin/flash']
```

Comment:
Please make this a relative symlink:

```python
postinstallcmds = ["cd %(installdir)s/bin && ln -s flash2 flash"]
```

Also, can we add a sanity check command like `flash --help` or whatever?

Reply:
done